pages/annotation-context/with-hotspot-inside-switch.page.tsx

```diff
@@ -76,26 +74,6 @@ export default function AnnotationScroll() {
                 { value: 'third', label: 'Third choice', description: 'This is a description' },
               ]}
             />
-
-            <Tiles
```

Comment:
We should remove the tutorial step in this case. Currently there are still 6 steps but only 5 are reachable on the page.